homeassistant/components/zwave_js/lock.py

```diff
@@ -0,0 +1,109 @@
+"""Representation of Z-Wave locks."""
+from enum import IntEnum
+import logging
+from typing import Any, Callable, Dict, List, Optional, Union
+
+from zwave_js_server.client import Client as ZwaveClient
+from zwave_js_server.const import CommandClass
+from zwave_js_server.model.value import Value as ZwaveValue
+
+from homeassistant.components.lock import DOMAIN as LOCK_DOMAIN, LockEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import STATE_LOCKED, STATE_UNLOCKED
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from .const import DATA_CLIENT, DATA_UNSUBSCRIBE, DOMAIN
+from .discovery import ZwaveDiscoveryInfo
+from .entity import ZWaveBaseEntity
+
+LOGGER = logging.getLogger(__name__)
+
+
+class DoorLockMode(IntEnum):
```

Comment:
This enum should probably be in the Python library.

https://github.com/home-assistant-libs/zwave-js-server-python/blob/d32c9dc4b933c30d5debb3b5af8b008643576ee0/zwave_js_server/const.py#L1